lib/internal/util/inspect.js

```diff
@@ -871,10 +831,9 @@ function formatRaw(ctx, value, recurseTimes, typedArray) {
       let bound = value;
       let fallback = '';
       if (constructor === null) {
-        const constr = findTypedConstructor(value);
-        fallback = constr.name;
+        fallback = TypedArrayPrototypeGetSymbolToStringTag(value);
```

Comment:
This should probably use:
```suggestion
      const fallback = TypedArrayPrototypeGetSymbolToStringTag(value);
      if (constructor === null) {
```

So that `fallback` is always non‑empty.

Reply:
I think the idea originally was to avoid the overhead of a function call where possible. Judging from our test coverage stats, constructor being `null` is the unusual case (616 times hitting the `if` but only 33 times evaluating to true).

The performance difference seems likely to be unmeasurable and the code simplification may very well be worth it. But I'd prefer that be handled in a separate pull request. I'd prefer that this pull request preserve the code shape and merely replace/eliminate the existing function.

This is reminding me to run benchmarks for this change.

Reply:
Well, `TypedArrayPrototypeGetSymbolToStringTag` is already invoked by `isTypedArray`.